webapp/lib/http_helpers.py

```diff
@@ -40,8 +31,8 @@ def error_response(error, file_path=""):
     413: E2BIG
     500: Anything else
     """
-
-    file_path = file_path or error.filename or ""
+    # We need getattr here because SwiftErrors don't have a filename attr
```

Comment:
But when [we're raising a SwiftError](https://github.com/canonical-web-and-design/assets.ubuntu.com/pull/146/files#diff-44ad1ff6e80390cf8eeea53b3d62bd47R222) aren't we passing through the file_path anyway?

Although actually, to think about the bigger picture, it might be much neater to move this code to a `@app.errorHandler(500)` decorated function and make it more intelligent so it can just natively process a bunch of errors without needing things passed through.

But maybe that ^ is out of scope for now.

Reply:
I made this change because I got an exception when developing. I would ideally want to move error handling to app.errorhandlers, but I would prefer to do it in a follow up PR.